site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -36,6 +36,26 @@ The same could be run for `alpha.swagger.json` and `beta.swagger.json` as requir
 
 You can read more about OpenAPI/Swagger code generation in their [Command Line Tool Documentation](https://swagger.io/docs/open-source-tools/swagger-codegen/)
 
+## SDK Functionality
+
+| Area            | Action              | Implemented |
+|-----------------|---------------------|-------------|
+| Lifecycle       | Ready               | ✔️          |
+| Lifecycle       | Health              | ✔️          |
+| Lifecycle       | Reserve             | ✔️          |
+| Lifecycle       | Allocate            | ✔️          |
+| Lifecycle       | Shutdown            | ✔️          |
+| Configuration   | WatchGameServer     | ✔️          |
+| Metadata        | SetAnnotation       | ✔️          |
+| Metadata        | SetLabel            | ✔️          |
+| Counters        | GetCounter          | ✔️          |
+| Counters        | UpdateCounter       | ✔️          |
+| Lists           | GetList             | ✔️          |
+| Lists           | UpdateList          | ✔️          |
+| Lists           | AddListValue        | ✔️          |
+| Lists           | RemoveListValue     | ✔️          |
+
```

Comment:
It also implemented player methods: GetPlayerCapacity, SetPlayerCapacity, PlayerConnect, GetConnectedPlayers, IsPlayerConnected, GetPlayerCount, PlayerDisconnect

Reply:
Labeled it as `Player Tracking` to maintain the uniformity. Please let me know if you would prefer `Player` instead.

Reply:
Yeah `Player Tracking` is good